src/vs/platform/workspace/common/workspace.ts

```diff
@@ -4,6 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 'use strict';
 
+import Event, { Emitter } from 'vs/base/common/event';
```

Comment:
never saw this import syntax. Is it the same as `import { default as Event, Emitter } from`?